exercises/pug/problem.ko.md

```diff
@@ -4,7 +4,7 @@ Pug 템플릿 엔진으로 렌더되는 홈페이지를 가지는 Express.js 앱
 
 뷰에서 'new Date.toDateString()'를 이용해 현재 시간을 표시해야 합니다.
 
-'toDateString()'를 사용해 간단히 시간을 재외하고 사람이 읽을 수 있는 형식으로
+'toDateString()'를 사용해 간단히 time형이 아닌 사람이 읽을 수 있는 형식으로
```

Comment:
This means `toDateString()` returns '2017-01-31' instead of '2017-01-31 03:08:00'.

How about:
> 'toDateString()'을 사용해 시간을 제외한, 사람이 읽을 수 있는 형식의 날짜를 반환할 수 있습니다.